pyaedt/modules/SetupTemplates.py

```diff
@@ -1,2380 +1,1500 @@
+        "Position": 0,
+        "SimSetupType": "kSIwaveDCIR",
+        "SimulationSettings": HFSS3DLayout_SimulationSettings,
+        "SweepDataList": HFSS3DLayout_SweepDataList,
+    }
+)
+
+HFSS3DLayout_LNASimulationSettings = OrderedDict(
+    {
+        "Enabled": True,
+        "GroupDelay": False,
+        "Perturbation": 0.1,
+        "Noise": False,
+        "Skip_DC": False,
+        "AdditionalOptions": "",
+        "BaseOptionName": "Default Options",
+        "FilterText": "",
+    }
+)
+LNA_Sweep = OrderedDict(
+    {
+        "DataId": "Sweep0",
+        "Properties": HFSS3DLayout_Properties,
+        "Sweep": SweepDefinition,
+        "SolutionID": -1,
+    }
+)
+HFSS3DLayout_LNAData = OrderedDict({"LNA Sweep 1": LNA_Sweep})
+LNA3DLayout = OrderedDict(
+    {
+        "Properties": HFSS3DLayout_Properties,
+        "CustomSetup": False,
+        "SolveSetupType": "LNA",
+        "Position": 0,
+        "SimSetupType": "kLNA",
+        "SimulationSettings": HFSS3DLayout_LNASimulationSettings,
+        "SweepDataList": HFSS3DLayout_SweepDataList,
+        "Data": HFSS3DLayout_LNAData,
+    }
+)
+MechTerm = OrderedDict(
+    {
+        "Enabled": True,
+        "MeshLink": meshlink,
+        "Solver": "Program Controlled",
+        "Stepping": "Program Controlled",
+    }
+)
 """Mechanical thermal setup properties and default values."""
 
-MechModal = [
-    ("Enabled", True),
-    ("MeshLink", meshlink),
-    ("Max Modes", 6),
-    ("Limit Search", True),
-    ("Range Max", "100MHz"),
-    ("Range Min", "0Hz"),
-    ("Solver", "Program Controlled"),
-]
+MechModal = OrderedDict(
+    {
+        "Enabled": True,
+        "MeshLink": meshlink,
+        "Max Modes": 6,
+        "Limit Search": True,
+        "Range Max": "100MHz",
+        "Range Min": "0Hz",
+        "Solver": "Program Controlled",
+    }
+)
 """Mechanical modal setup properties and default values."""
 
-MechStructural = [
-    ("Enabled", True),
-    ("MeshLink", meshlink),
-    ("Solver", "Program Controlled"),
-    ("Stepping", "Program Controlled"),
-]
+MechStructural = OrderedDict(
+    {
+        "Enabled": True,
+        "MeshLink": meshlink,
+        "Solver": "Program Controlled",
+        "Stepping": "Program Controlled",
+    }
+)
 """Mechanical structural setup properties and default values."""
 
 # TODO complete the list of templates for other Solvers
 
-RmxprtDefault = [
-    ("Enabled", True),
-    ("OperationType", "Motor"),
-    ("LoadType", "ConstPower"),
-    ("RatedOutputPower", "1kW"),
-    ("RatedVoltage", "100V"),
-    ("RatedSpeed", "1000rpm"),
-    ("OperatingTemperature", "75cel"),
-]
+RmxprtDefault = OrderedDict(
+    {
+        "Enabled": True,
+        "OperationType": "Motor",
+        "LoadType": "ConstPower",
+        "RatedOutputPower": "1kW",
+        "RatedVoltage": "100V",
+        "RatedSpeed": "1000rpm",
+        "OperatingTemperature": "75cel",
+    }
+)
 """RMxprt Default setup properties and default values."""
```

Comment:
I thought there was an early suggestion that capitalized the "X" for RMXprt--but I may be wrong. Based on this content, it doesn't  seem like it should be capitalized.